BraveWallet/Preview Content/MockContent.swift

```diff
@@ -39,13 +39,13 @@ extension BraveWallet.BlockchainToken {
     coin: .eth
   )
   
-  static let mockSolToken: BraveWallet.BlockchainToken = .init(
+  static let mockSPLToken: BraveWallet.BlockchainToken = .init(
```

Comment:
Can we keep this mock token? I ended up [using it in Solana unit tests](https://github.com/brave/brave-ios/commit/712c1757f0ef077427d46923b6f5c3c6f495e6f3#diff-9e9e608268bf47d47393fff853d30f9779e66942952910382a47dba7700934d2R72) (and set `contractAddress` to empty string), and added a new `mockSpdToken`